doc/rados/deployment/ceph-deploy-new.rst

```diff
@@ -45,8 +45,14 @@ Naming a Cluster
 ----------------
 
 By default, Ceph clusters have a cluster name of ``ceph``. You can specify
-a cluster name if you want to run multiple clusters on the same hardware. For
-example, if you want to optimize a cluster for use with block devices, and
+a cluster name if you want to run multiple clusters on the same hardware. 
```

Comment:
I think it would be safer to just omit the whole `--cluster` thing, because it hasn't been supported for a while and it is actively discouraged. Mentioning a certain commit from ceph-deploy might imply that a previous version would work, but there are parts of Ceph where it will not work at all if a cluster name different than `ceph` is used.

Reply:
Hi, @alfredodeza, @gregsfortytwo
Thanks for the attention!

Yeah, I also personally think removing that paragraph would be much simpler